Tests/PcppTestFramework/PcppTestFramework.h

```diff
@@ -15,6 +15,13 @@
 #define ptr_PTF_PRINT_TYPE_EXPECTED(exp, val) exp << "[ptr: " << val << "]"
 #define enumclass_PTF_PRINT_TYPE_ACTUAL(exp, val) "enum[" << +static_cast<std::underlying_type<decltype(val)>::type>(val) << "]"
 #define enumclass_PTF_PRINT_TYPE_EXPECTED(exp, val) exp << "[" << +static_cast<std::underlying_type<decltype(val)>::type>(val) << "]"
+#define BUFFER_PRINT(buffer, size) \
```

Comment:
could you provide an example of the log by this function?

Reply:
Sure.

```
TcpPacketCreation                  : FAILED (/home/wmatusiak/private/PcapPlusPlus/Tests/Packet++Test/Tests/TcpTests.cpp:263). Assert BUFFER COMPARE failed:
   Actual   
08 00 27 19 1c 78 30 46 9a 23 fb fa 08 00 45 00 00 41 4f 4c 40 00 3b 06 47 94 d4 c7 ca 09 0a 00 00 06 00 50 ac 80 b8 29 cb 98 e9 77 15 86 90 18 4e d2 de a3 00 00 01 01 08 0a d3 f0 39 98 00 03 00 95 01 03 03 00 00 49 45 4e 44 ae 42 60 82 
   Expected   
08 00 27 19 1c 78 30 46 9a 23 fb fa 08 00 45 00 00 41 4f 4c 40 00 3b 06 47 94 d4 c7 ca 09 0a 00 00 06 00 50 ac 80 b8 29 cb 98 e9 77 15 86 90 18 4e d2 de a1 00 00 01 01 08 0a d3 f0 39 98 00 03 00 95 01 03 03 02 00 49 45 4e 44 ae 42 60 82
```